src/pyedb/dotnet/edb_core/edb_data/layer_data.py

```diff
@@ -705,3 +725,35 @@ def _load_layer(self, layer):
                     layer["side_hallhuray_surface_ratio"],
                     apply_on_surface="side",
                 )
+
+    @property
+    def properties(self):
+        data = {}
+        data["name"] = self.name
+        data["type"] = self.type
+        data["material"] = self.material
+        data["fill_material"] = self.fill_material
+        data["thickness"] = self._edb_object.GetThicknessValue().ToString()
+        data["color"] = self.color
+        return data
+
+    @properties.setter
+    def properties(self, params):
```

Comment:
Same remark as before.

Reply:
dict.get() returns None if key doesn't exist, so no exception will be triggered.